distribution/docs/src/main/resources/_contents/_tables/ddf.catalog.registry.policy.RegistryPolicyPlugin-table-contents.adoc

```diff
@@ -0,0 +1,55 @@
+.[[ddf.catalog.registry.policy.RegistryPolicyPlugin]]Registry Policy Plugin
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Registry CUD Attributes
```

Comment:
This field has been broken up into three separate ones so you can controll create/update/delete individually.
Here is the info from the metatype
```
<AD description="Roles/attributes required for create operations on registry entries. Example: {role=role1;type=type1}"
            name="Registry Create Attributes" id="createAccessPolicyStrings" required="true"
            type="String" cardinality="100"
            default="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=guest"/>

        <AD description="Roles/attributes required for update operations on registry entries. Example: {role=role1;type=type1}"
            name="Registry Update Attributes" id="updateAccessPolicyStrings" required="true"
            type="String" cardinality="100"
            default="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=guest"/>

        <AD description="Roles/attributes required for delete operations on registry entries. Example: {role=role1;type=type1}"
            name="Registry Delete Attributes" id="deleteAccessPolicyStrings" required="true"
            type="String" cardinality="100"
            default="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=guest"/>
```